pkg/apis/autoscaling/v1alpha1/pa_defaults.go

```diff
@@ -30,19 +41,43 @@ func (r *PodAutoscaler) SetDefaults() {
 		// Default class to KPA.
 		r.Annotations[autoscaling.ClassAnnotationKey] = autoscaling.KPA
```

Comment:
🤔 I feel like this is possibly the most important thing to default from the configmap.

Reply:
cc @tcnghia since Networking has such a configmap where we could do this.

Reply:
Absolutely, we should have a default class in the config map.  I'll add that after this pr lands.